executor/oci/hosts.go

```diff
@@ -76,3 +70,14 @@ func makeHostsFile(stateDir string, extraHosts []executor.HostIP, idmap *idtools
 		os.RemoveAll(p)
 	}, nil
 }
+
+func initHostsFile(hostname string) string {
```

Comment:
Wondering; is validation needed on the value? (i.e., would `google.com` or any other FQDN be allowed?)

Reply:
@thaJeztah I checked the code in docker builder and buildah, but not found similar code logic, chould you please share me some hints on how to handle it? Or just leave it here when we meet with that scarios?